ChangeLog.d/add_final_delay_accessor

```diff
@@ -1,4 +1,4 @@
 Features
    * Add the function mbedtls_timing_get_final_delay() to access the private
      final delay field in an mbedtls_timing_delay_context, as requested in
-     #5183
+     #5183.
```

Comment:
Rather than have to do this check manually, why not update `assemble_changelog.py` to add a trailing full stop if one is missing? It could also reflow the text if that is desired.

Reply:
And if we're going to change this, how about removing `the` for consistency?

`Add function mbedtls_timing_get_final_delay()...`

Reply:
We usually write changelog entries in proper English, not headlinese, thus “add a function” or (since it's named) “add the function”.

Regarding automation: automation has the cost of writing the code and of handling exceptions. Exceptions are why we aren't reflowing (sometimes we want forced breaks at specific points) and why we aren't doing spell checking. I'm not sure if there are exceptions where we really don't want a full stop at the end.

Reply:
The ChangeLog is presented as a bulleted list. A bulleted list is not really fully prose, in the same way documentation is; and changelogs are typically read rapidly by users to identify "what might interest me in this new release". So writing for rapid parsing (i.e. consistency across multiple entries) and removal of filler would seem to be reasonable here.

Generally in text, where bulleted lists appear they will (depending on style) either (1) all but the last item end in `;`, last item ends in `.` (this is common in documents where bulleted lists aren't the main content) or (2) none have punctuation at the end (which is common on slides).

However, this gets tricky when some bullet points might be multiple sentences, because each sentence within a bullet point needs its punctuation at the end, and so (for consistency) the last sentence in a multi-sentence bullet point should have its punctuation, and so (also for consistency) a bullet point of only one sentence should also have punctuation at the end.

One might consider not doing this for a very short phrase, such as `Fixes #1234`, but since we do put full stops at the end of those, I can't see content that wouldn't have punctuation at the end.